src/engine/Settings.cs

```diff
@@ -340,6 +340,12 @@ private Settings()
     public SettingValue<JSONDebug.DebugMode> JSONDebugMode { get; private set; } =
         new(JSONDebug.DebugMode.Automatic);
 
+    /// <summary>
+    ///   The screen effect currently being used
+    /// </summary>
+    [JsonProperty]
+    public SettingValue<ScreenEffect?> CurrentScreenEffect { get; private set; } = new(null);
```

Comment:
Should the screen effect be in graphics tab? It is after all a graphical thing.

Reply:
While it is graphical, I was trying to keep more silly options to misc. Your choice though.

Reply:
I guess then we can put it in the misc tab if there's a chance of there being non-graphical joke options.